dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -358,10 +358,14 @@ $small-footer-standard-width: 400px;
   position: absolute;
   bottom: 0;
   left: 0;
-  width: 625px;
+  width: 650px;
```

Comment:
It was... but it wasn't after I started using the inner scroll area.  Hm.  To be honest I'm not sure now which dimension changed to cause it to start wrapping, but it bothered me, so I made the popout wider.